docs/admin/kube-apiserver.md

```diff
@@ -72,6 +72,7 @@ kube-apiserver
       --cors-allowed-origins=[]: List of allowed origins for CORS, comma separated.  An allowed origin can be a regular expression to support subdomain matching.  If this list is empty CORS will not be enabled.
       --delete-collection-workers=1: Number of workers spawned for DeleteCollection call. These are used to speed up namespace cleanup.
       --deserialization-cache-size=50000: Number of deserialized json objects to cache in memory.
+      --enable-garbage-collector[=false]: Enables the generic garbage collector. MUST be synced with the corresponding flag of the kube-controller-manager.
```

Comment:
Do you want to say something about this being an alpha feature and admins should be careful and understand the risks before setting this to true? :)